### ISSUE TITLE
Filter Table taking Default values into consideration

### ISSUE THREAD

**Issue by Akhliskun** [Akhliskun/taskcluster-worker-checker#125]
This issue is for the GUI branch.

Currently we show all the machines if we don't have IDLE and/or Ignored? selected.
By default the filter logic should go as described below, even if nothing is selected (IDLE/Ignored) in the UI:

1. Is Machine Ignored? **Yes? Don't show it in the table**. No? Show it.
2. Is the machine IDLE for 6 Hours or more? **Yes? Show it in the table**. No? Don't show it.

Basically we are missing "default values" which will be re-written by the custom input that the user sets in the UI.

**Comment by bolchisb** [comment]
The purpose of having checkboxes is to display data unfiltered. Checking a box will activate the filter. I will set the filters to use defaults values when activated and still having the possibility to change filter values.

**Comment by bolchisb** [comment]
Another case of incomplete list.
('t-linux64-ms-539', 't-linux64-ms-539')
('t-linux64-ms-540', 't-linux64-ms-540')
('t-linux64-ms-571', 't-w1064-ms-016')
('t-linux64-ms-572', 't-w1064-ms-017')
('t-linux64-ms-573', 't-w1064-ms-018')
('t-linux64-ms-574', 't-w1064-ms-019')
('t-linux64-ms-575', 't-w1064-ms-020')
('t-linux64-ms-576', 't-w1064-ms-021')
('t-linux64-ms-577', 't-w1064-ms-022')
('t-linux64-ms-578', 't-w1064-ms-023')
('t-linux64-ms-579', 't-w1064-ms-024')
('t-linux64-ms-580', 't-w1064-ms-025')
('t-w1064-ms-016', 't-w1064-ms-026')
('t-w1064-ms-017', 't-w1064-ms-027')
('t-w1064-ms-018', 't-w1064-ms-028')

**Comment by bolchisb** [comment]
Issue solved, improved search and filter method, added a new function to the Machine class, it's looking for the idle time for each machine. You can start QA testing.
